### DISCUSSION TITLE
Transferring ownership of a project to another organization

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#6681] [Feature Requests]
# Update - Shipped 🚢 

Project transfers across organizations has been shipped! 🚢 

For the first iteration of project transfers, you do need to at least be a member of the target organization (the organization you want to transfer the project to). You can read additional docs [here](https://supabase.com/docs/guides/platform/project-transfer).

## Is your feature request related to a problem? Please describe.

Currently, it is not possible to transfer your project to other users. This can be very useful in case we test things on a database, and then want to use it in production.

## Describe the solution you'd like

A simple modal where we would just have to specify the email of the new owner, and confirm by email our action (or reconnect). Something similar to Github.

## Describe alternatives you've considered

/

## Additional context

Many developers ask for this feature (e.g from @netgfx, [last August](https://github.com/supabase/supabase/discussions/2814_)), and it is not yet tracked, which is why i created this issue.

**Comment by odiseo0** [comment]
I like this

**Comment by MildTomato** [comment]
There are some reasons we haven't been able to roll this out as fast as we would of liked.
Mostly due to whether or not a project is 'paid for' and is incurring additional usage costs.

We want to make sure transferring a project is seamless to another organization, and that we can easily roll up and finalize any outstanding billing before moving it to the new organization.

I've renamed the title to refer to 'organization' as well since currently, we adhere to an "organization" structure, which contains "projects" and a user can be the owner of an organization. 

For security purposes, we'd perhaps have to only allow transfer of projects between projects you are at least a member of.

**Comment by bradgreens** [comment]
Another example of how someone ends up needing a transfer. AFAIK Supabase requires only Github auth. I use my Github account for  personal repos and it is connected to my Company's Github organization. 

I initially set a project up for my Company this way, but later realized I do not want to mix personal Supabase orgs/projects with my Company ones (I'm not sure if this may be a SOC 2 compliance issue, which we're in the process of achieving).

**Comment by kaufmann42** [comment]
Any updates on this?

**Comment by yenicelik** [comment]
Would also love to see this

**Comment by bergiusj** [comment]
Would love to see this feature as well. I totally understand the complications regarding projects on a paid plan. But how about moving one project between my own organizations, where both are on a free plan!?

**Comment by mosnicholas** [comment]
+1 would be super useful

**Comment by itsmaleen** [comment]
+1 would really love this feature

**Comment by ioucyf** [comment]
+1 Please make it happen. 🙏🏼

**Comment by nexbotai** [comment]
I'm surprised this is still not a thing. Please

**Comment by crstnmac** [comment]
+1 please make it happen 🥹

**Comment by RyanLuttrell** [comment]
Yes please! +1

**Comment by iam-rohid** [comment]
I'm working on a client app for 2 months. I created the project on my own org and now I want to hand over the project to my client and just realizing that you can't do that! 😕. I have lots of other projects in my own org and I can't give access to my own org to my client.

**Comment by BenjaminGrabow** [comment]
I would love that feature too! +1

**Comment by ArkadyBuryakov** [comment]
I also await for this feature. It would make life a lot easier.

**Comment by kaelansmith** [comment]
Me too! Facing the same issue where I started my client project in my own account and now wish to transfer to client's own account.

**Comment by stargt** [comment]
Any updates?

**Comment by jamesthesken** [comment]
This would be a great feature!

**Comment by adamkudz** [comment]
I am in the same situation.  A solution would be great! Please?

**Comment by siddarthvader** [comment]
+1

**Comment by kevcodez** [comment]
Happy to announce that project transfers to a different organization are coming next week!

This is one of the neat features that are possible due to some groundwork changes to our billing - please read the [full announcement](https://supabase.com/blog/organization-based-billing).

There's also a [GitHub discussion](https://github.com/orgs/supabase/discussions/17061) around the announcement, in case you have follow-up questions regarding the billing changes.

For the first iteration of project transfers, you do need to at least be a member of the target organization (the organization you want to transfer the project to). You can read additional docs [here](https://supabase.com/docs/guides/platform/project-transfer).

**Reply by kevcodez** [reply]
Project Transfers are now generally available :)
